src/bosquet/generator.clj

```diff
@@ -22,7 +25,7 @@
   and set it as required inputs for the resolver.
   For the output check if themplate is producing generated content
   anf if so add a key for it into the output"
-  [the-key template]
+  [the-key template model-opts]
```

Comment:
the new map of model-opts which will be merged with the model opts defined in template definition

In practice these are "llm-generate-opts" as used for any "llm-generate" tag.